websocketpp/extensions/permessage_deflate/enabled.hpp

```diff
@@ -741,6 +805,7 @@ class enabled {
     int m_flush;
     size_t m_compress_buffer_size;
     lib::unique_ptr_uchar_array m_compress_buffer;
+    lib::unique_ptr_uchar_array m_decompress_buffer;
```

Comment:
Why add this separate buffer?

Reply:
https://github.com/zaphoyd/websocketpp/commit/6330286ad5d56b918d8e32d32cb5860ee4619359